CMakeLists.txt

```diff
@@ -260,6 +261,20 @@ IF(WANT_PORTAUDIO)
 	ENDIF(PORTAUDIO_FOUND)
 ENDIF(WANT_PORTAUDIO)
 
+# check for libsoundio
+IF(WANT_SOUNDIO)
+  FIND_PACKAGE(SoundIo)
+  IF(SOUNDIO_FOUND)
+    SET(LMMS_HAVE_SOUNDIO TRUE)
+    SET(STATUS_SOUNDIO "OK")
+    INCLUDE_DIRECTORIES("${SOUNDIO_INCLUDE_DIR}")
```

Comment:
Not sure how big of a deal this is, but all the other audio/midi backends configure their includes in `/src/CMakeLists.txt` instead of `/CMakeLists.txt` so as not to affect the building of plugins.